examples/vae.py

```diff
@@ -123,9 +123,9 @@ def model_sample(self, batch_size=1):
         # sample the handwriting style from the constant prior distribution
         prior_mu = Variable(torch.zeros([batch_size, self.z_dim]))
         prior_sigma = Variable(torch.ones([batch_size, self.z_dim]))
-        zs = pyro.sample("z", dist.Normal(prior_mu, prior_sigma).reshape(extra_event_dims=2))
+        zs = pyro.sample("z", dist.Normal(prior_mu, prior_sigma))
```

Comment:
Why don't you need to `.reshape(extra_event_dims=1)` here?

Reply:
From my understanding, `model_sample` is not used for inference but for generating samples separately using the trained `decoder`, for plotting, etc.